schema/elasticsearch/visibility/versioned/v8/index_template_v7.json

```
@@ -0,0 +1,102 @@
+{
+  "order": 0,
+  "index_patterns": ["temporal_visibility_v1*"],
+  "settings": {
+    "index": {
+      "number_of_shards": "1",
+      "number_of_replicas": "0",
+      "auto_expand_replicas": "0-2",
+      "search.idle.after": "365d",
+      "sort.field": ["CloseTime", "StartTime", "RunId"],
+      "sort.order": ["desc", "desc", "desc"],
+      "sort.missing": ["_first", "_first", "_first"]
+    }
+  },
+  "mappings": {
+    "dynamic": "false",
+    "properties": {
+      "NamespaceId": {
+        "type": "keyword"
+      },
+      "TemporalNamespaceDivision": {
+        "type": "keyword"
+      },
+      "WorkflowId": {
+        "type": "keyword"
+      },
+      "RunId": {
+        "type": "keyword"
+      },
+      "WorkflowType": {
+        "type": "keyword"
+      },
+      "StartTime": {
+        "type": "date_nanos"
+      },
+      "ExecutionTime": {
+        "type": "date_nanos"
+      },
+      "CloseTime": {
+        "type": "date_nanos"
+      },
+      "ExecutionDuration": {
+        "type": "long"
+      },
+      "ExecutionStatus": {
+        "type": "keyword"
+      },
+      "TaskQueue": {
+        "type": "keyword"
+      },
+      "TemporalChangeVersion": {
+        "type": "keyword"
+      },
+      "BatcherNamespace": {
+        "type": "keyword"
+      },
+      "BatcherUser": {
+        "type": "keyword"
+      },
+      "BinaryChecksums": {
+        "type": "keyword"
+      },
+      "HistoryLength": {
+        "type": "long"
+      },
+      "StateTransitionCount": {
+        "type": "long"
+      },
+      "TemporalScheduledStartTime": {
+        "type": "date_nanos"
+      },
+      "TemporalScheduledById": {
+        "type": "keyword"
+      },
+      "TemporalSchedulePaused": {
+        "type": "boolean"
+      },
+      "HistorySizeBytes": {
+        "type": "long"
+      },
+      "BuildIds": {
+        "type": "keyword"
+      },
+      "ParentWorkflowId": {
+        "type": "keyword"
+      },
+      "ParentRunId": {
+        "type": "keyword"
+      },
+      "RootWorkflowId": {
+        "type": "keyword"
+      },
+      "RootRunId": {
+        "type": "keyword"
+      },
+      "TemporalPausedInfo": {
```

Please fix typo here and other places.
```suggestion
      "TemporalPauseInfo": {
```